mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -91,6 +92,14 @@
 )
 
 
+def load_model_with_mlflow_config(model_uri):
```

Comment:
Can we update pyfunc load_model to detect environment variable SERVING_MODEL_CONFIG if model_config is None? So that we don't need this method here

Reply:
we can . then we should rename the environment variable to `MODEL_CONFIG` (not only for serving)

Reply:
So in this PR I added `SERVING_MODEL_CONFIG` just for launching the model server which is only for serving this model.

thoughts ? @serena-ruan

Reply:
Yea that makes sense, let's keep this method here.
For databricks serving, we need to consider the CUJ and how to make this work from UI. e.g. is it convenient for users to pass the model_config as a json string, or is a path directing to the model config file better? We also need to update databricks serving code to match this one, and check MLflow version (follow-up work needed)

Reply:
yes this is follow-up work. I'll create backlog task